mnist.py

```diff
@@ -146,13 +146,18 @@ def model_fn(features, labels, mode, params):
         # Save accuracy scalar to Tensorboard output.
         tf.summary.scalar('train_accuracy', accuracy[1])
 
+        tf.summary.scalar('loss', loss)
+
```

Comment:
Why we are saving scalar here if loss is not yet calculated?

Reply:
Here we only initialize. I also changed scalar loss to eval_loss in line 159, because it was two loss values and it could be misunderstooded.